### PR TITLE
Cu 7mr5pa logins page

### PR DESCRIPTION
**Pull request by Hamza-A**
This branch aims to improve the logins page by adding all the components it needs to look close to our wireframe

## Description
The goal of this branch is to make the `/account/logins` page as close as possible to our wireframe by:
- adding a button beneath email list and phone list to hide/show non primary identities. To do this, I have created a new styled component `<ShowMoreButton>` which appears only if the corresponding identity list > 1
- adding "social logins" category which doesn't has any content for now
- changing the "add new [identity]" button to make it redirecting to `/account/logins/[type]/new`, for now this is a blank page that should be implemented in another branch
- fixing the text at the header to make the following text appear "Your emails, phones and social logins"

## How Has This Been Tested?

A whole `describe` with 3 tests has been added to test the `<ShowMoreButton>`'s behaviour. Since this button has been added to the main page of the app, the `describe` is located at `tests/pages/account.test.tsx`

## Screenshots

image

## Types of changes

- Chore (non-breaking change which refactors / improves the existing code base)
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to
  change)

## Checklist:

- :green_circle: My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- :green_circle: I have read the [**CONTRIBUTING**][contributing_file] document.
- :green_circle: I have added tests to cover my changes.
- :green_circle: All new and existing tests passed.

[contributing_file]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-7mr5pa Logins page](https://app.clickup.com/t/7mr5pa)

File: src/pages/account/logins/index.tsx [hunk 1]
```diff
@@ -28,8 +26,25 @@ type LoginsProps = {
...
+  );
+  const [hideSecondaryPhones, setHideSecondaryPhones] = React.useState<boolean>(
+    true,
+  );
+
+  let emailList;
+  let phoneList;
+
+  hideSecondaryEmails
+    ? (emailList = sortedIdentities.emailIdentities.filter((id) => id.primary))
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:40]: Using `id` is misleading in this context.
```suggestion
    ? (emailList = sortedIdentities.emailIdentities.filter((identity) => identity.primary))
```

File: src/pages/account/logins/index.tsx [hunk 2]
```diff
@@ -28,8 +26,25 @@ type LoginsProps = {
...
+    : (emailList = sortedIdentities.emailIdentities);
+
+  hideSecondaryPhones
+    ? (phoneList = sortedIdentities.phoneIdentities.filter((id) => id.primary))
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:44]:
```suggestion
    ? (phoneList = sortedIdentities.phoneIdentities.filter((identity) => identity.primary))
```

File: src/pages/account/logins/index.tsx [hunk 3]
```diff
@@ -232,3 +249,24 @@ const Button = styled.button<ButtonProps>`
     color: ${({ theme }) => theme.colors.contrastCopy};
   }
 `;
+
+export const ShowMoreButton = styled.button<ButtonProps>`
+  padding: 0.5rem;
+  margin-right: 1rem;
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:255]: Use spacing values from the theme.

File: src/pages/account/logins/index.tsx [hunk 4]
```diff
@@ -232,3 +249,24 @@ const Button = styled.button<ButtonProps>`
...
+  border-radius: ${({ theme }) => theme.radii[0]};
+  background-color: transparent;
+  ${(props) => `color: ${props.color}`};
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:258]: This is taken from my previous button. You don't need to pass the color prop any more, just use the theme right away :).

Let me know if this is not clear.

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/pages/account/logins/index.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: OK fixed 0fe5a7d

File: src/pages/account/logins/index.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: OK fixed 0fe5a7d

File: src/pages/account/logins/index.tsx (hunk 4 above)
Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: OK fixed 0fe5a7d

File: src/pages/account/logins/index.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: We're not using theme anymore in this file

File: src/pages/account/logins/index.tsx [hunk 5]
```diff
@@ -208,16 +222,33 @@ export const Value = styled.p`
...
+  padding: 0.5rem;
+  margin-right: 1rem;
+  border-radius: ${({ theme }) => theme.radii[0]};
+  background-color: transparent;
+  color: ${({ theme }) => theme.colors.green};
+  transition: ${({ theme }) => theme.transitions.quick};
+  font-weight: ${({ theme }) => theme.fontWeights.medium};
+  &:hover {
+    cursor: pointer;
+    ${(props) => `background-color: ${props.color}`};
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:235]: You forgot the ones for `background-color`.

Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: Oops, fixed e6d1841

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**